draft responses to Lakens (#99)
aaronpeikert/bayes-prereg: reviews.Rmd

In the following we address your and each reviewers'/editors' concerns point-by-point.
For Daniel Laken's review we took the liberty to additionally include a high level summary, since we found it, at times, hard to follow the different interwoven discussions.

All changes of the manuscript since this review can be viewed on GitHub:

[https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main](https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main#diff-a9a4aad3fa8c9c10c5404b632bc3a01a25d2d8430eb932bc35c76769963e4b70)

# Daniel Lakens

:::reviewer
Reviewer #1: The authors set out to explain what the primary aim of preregistration is.
I think they are correct that many scientists have hardly thought about this question.
This is not in itself bad - preregistration intuitively makes sense - but it is important to clearly articulate what the aim of preregistration is.
:::

We agree.

:::reviewer
So I applaud the authors for believing this is important.
At the same time, their arguments were, I believe, not always equally convincing, and they do not adequately discuss the existing literature.
:::

> We thank the reviewer for pointing out that we failed to include opposing philosophical perspectives.
> We are now citing Laken's and Mayo's positions and relate them to our perspective.

Reviewer: ```suggestion
We thank the reviewer for pointing out that we did not include perspectives that disagree on a very fundamental level about the philosophical foundations of empirical science.
We are now discussing Laken's and Mayo's positions.
```

:::reviewer
The main argument of the authors is straightforward: preregistration reduces uncertainty about the Type 1 error rate, and under the assumption that it is possible to inflate the Type 1 error rate when a study is not preregistered, reducing the uncertainty through preregistration increases the posterior probability of the hypothesis after a test that yields a significant result.
The authors say they provide a 'Bayesian rationalization'.
This is not correct.
:::

We respectfully disagree.
What makes the arguments "Bayesian" is our interpretation of probabilities.
For a Bayesian, they represent degrees of belief.
For a frequentist, they represent (relative) frequencies of events that result from a random process.
This is a subtle but most important distinction.

:::reviewer
Their rationalization is in essence based on the likelihood ratio.
All else equal, a lower Type 1 error rate means that observing a significant result is relatively more likely when H1 is true.
Of course a higher likelihood ratio also means a higher Bayes factor, which means a higher probability that H1 is true in a Bayesian framework.
:::

> It is well known that the math for frequentists is similar or even identical to Bayesian under the right conditions.

Reviewer: ```suggestion
It is well known that the computations of frequentist tests can be similar or even identical to Bayesian approaches under certain conditions.
```

Still, a frequentist would not conclude anything about the probability of H1.
However, our goal is to say something about how preregistration changes how believable researchers should find claims made in the literature.
This necessitates a Bayesian perspective on the process of scientific inquiry (which we adopted) yet it applies to cases of researchers either using frequentist and Bayesian inference in their daily routine.

:::reviewer
But the workhorse in the rationalization that the authors provide is the likelihood ratio.
This does not change when we put a distribution on the Type 1 error rate (as the authors do).
:::

Yes, the Bayes formula can be conceptualized as a likelihood ratio scaled by priors.

> However, the mere use of the likelihood ratio in the Bayes theorem can not be grounds to reduce Bayesianism to likelihood ratios.

Reviewer: Very good response.

:::reviewer
The distribution (or the move to a Bayesian rationalization) does not add anything (although of course it can be added).
:::

We find that it adds value by making an otherwise implicit process (that is, the process of updating beliefs in the theories) explicit to the point of making it calculable.
To our knowledge, there is no other perspective in the literature that makes this part of the process explicit.
By conceptualising uncertainty with a specific distribution, we rid ourselves of vague notions of transparency, control, or exploration.
This is of utmost importance because only this clarity enables us to effectively navigate the tradeoffs involved.

:::reviewer
Importantly, the likelihood function of course also underlies the uniformly most powerful test in a Neyman-Pearson hypothesis testing framework.
So for a frequentist (and I mean a frequentist not in statistical terms, but also in Popperian anti-Bayesian view on confirmation) the same argument for preregistration plays a role.
:::

> We agree that this argument could perhaps be made for a frequentist as well.

Reviewer: Important points, but maybe better for the high-level summary. So for this point, I would just add a short answer like "Yes, you could probably make a similar argument, but 1. we are not concerned with that 2. specific tradeoffs are hard". Then, in the high-level summary you could write about this point "Faced with a specific tradeoff, e.g., is a particular post hoc change of a preregistration beneficial, we must revert to vague notions and "gut" instinct under this philosophy.
Within the Bayesian framework, we may directly ask: What consequences should preregistration have on the beliefs of individual scientists or the research community as a whole?".

Mayo's severity perspective is taking one more step in this direction by asserting that belief in a hypothesis is justified to the extent that it survives stringent scrutiny.
However, even this "strong" severity principle is only a proportional statement.
Faced with a specific tradeoff, e.g., is a particular post hoc change of a preregistration beneficial, we must revert to vague notions and "gut" instinct under this philosophy.
Within the Bayesian framework, we may directly ask: What consequences should preregistration have on the beliefs of individual scientists or the research community as a whole?

:::reviewer
The title should not read a Bayesian rationalization, but a likelihood rationalization.
A higher likelihood is not always better, as one can cherry pick a high likelihood ratio.
This is extensively discussed by Mayo (2018).
After p-hacking, we could get a very high likelihood (or Bayes factor) that supports H1 - but it was cherry-picked, and is highly likely to be a fluke.
Therefore, the authors limit themselves to relative likelihoods.
:::

We did not limit ourselves to relative liklihoods.

> We explicitly include both the "firmness" and "increase in firmness" perspective.
> That a firmness perspective has significant flaws is of no matter to our argument since we intend to show that someone who subscribes to either must be more persuaded by a preregistered study.

Reviewer: ```suggestion
We explicitly include both the "firmness" and "increase in firmness" perspective.
Since we showed that out justification of preregistration holds for both perspectives, the discussion about specific flaws of the firmness perspective do not concern our argument.  
```

Author: added

:::reviewer
They point out that, assuming people only increase their alpha level when p-hacking, preregistration leads to tests with a lower alpha level.
As such, they basically restate Mayo's (2018) argument about the severity of a test.
:::

> Let us cite <https://doi.org/10.3758/s13423-022-02069-1"Nowhere in Mayo’s severity principle or in the specifications discussed below is the riskiness or specificity of a hypothesis, and the predictions it makes, incorporated as a requirement for a severe test (Meehl, 1978; 1986; 1990b, a; Roberts & Pashler, 2000)."

Reviewer: Some formatting issues.

:::reviewer
However, they never make this explicit, and I think this is to the detriment of the paper.
:::

We added a new section that deals with the concept of severity, making it explicit how our approach differs fundamentally.

:::reviewer
The fact that they subsequently add a Bayesian sauce to the likelihood ratio (by adding a distribution to the Type 1 error rate when a study is not preregistered, instead of simply making some assumption about a point-value of the inflated Type 1 error rate) does not add anything to their argument.
So, I believe the authors have basically attempted to re-invent Mayo's severity argument.
:::

The whole second part of the paper deals with the problem of not being able to reduce the theoretical risk to a point estimate.
The statistical procedure has an objective Type 1 error rate, for which we can come up with a point estimate if we agree on the conditions, which is a starting point for evaluating the theoretical risk of the scientific (!= statistical) procedure.

> Since, a reader does not have access to the whole scientific procedure, they must judge it with uncertainty.
> We find it important to distentengle the statistical procedure, the scientific procedure, and the outside perspective of the reader.

Reviewer: I wonder if that would also be his review on Bayesian Textbooks: "Why don't they just use a point estimate, no need for this bayesian sauce" ^^. Adding a distribution to quantify uncertainty is just the correct mathematical formulation. Moving around point estimates is not.
```suggestion
Since readers do not have access to the whole scientific procedure, they must judge it with uncertainty.
We find it important to disentangle the statistical procedure, the scientific procedure, and the outside perspective of the reader.
```

:::reviewer
The fact that this is not immediately clear is mainly due to missing discussions of the existing literature.
Point 1: Preregistration can not objectively minimize uncertainty about the Type 1 error rate The main argument of the authors is that risky predictions create persuasive evidence if they turn out to be correct.
Preregistrations are ideal for communicating the theoretical risk researchers took when testing a claim, and communicating this removes uncertainty in the reader about the theoretical risk the authors took.
I think this argument is in the right direction, but it is not fully developed.

> I was somewhat disappointed not to see a discussion of my own paper on a conceptual analysis of the value of preregistration, as I had a similar goal as the authors when I wrote my paper, and my arguments are in the same direction, but dare I say, slightly better developed.

Reviewer: ```suggestion
```

:::

We find ourselves in a curios position, that perhaps may be summarised as "doing the right thing, for the wrong reasons", i.e., we agree with Laken's conclusions about preregistration but we disagree how to come to these conclusions.
We take a Bayesian/inductionist view, whereas Lakens takes a frequentist/anti-inductionist view, agreeing on the end result but disagreeing on everything that comes before.
While we find it important that readers are aware, how anti-inductionist deal with preregistration, this paper was not meant as an exercise to compare Bayesian justification for preregistration with its frequentist counterpart.

:::reviewer

> Maybe everyone thinks this of their own papers, so forgive me if I am too biased, but I will attempt to point out some weaknesses in the current paper, sometimes by citing my own paper.

Reviewer: ```suggestion
I was somewhat disappointed not to see a discussion of my own paper on a conceptual analysis of the value of preregistration, as I had a similar goal as the authors when I wrote my paper, and my arguments are in the same direction, but dare I say, slightly better developed.
Maybe everyone thinks this of their own papers, so forgive me if I am too biased, but I will attempt to point out some weaknesses in the current paper, sometimes by citing my own paper.
```

I hope the authors see how strongly the ideas are related, and forgive me for citing myself.
:::

> Of course, we see the relevance of Laken's work; we met in Eindhoven to discuss an early version of the paper.

Reviewer: With respect to how well developed our respective arguments are, we would like to point out that we think that a key part of our argument is the mathematical formulation that enables research to actually compute tradeoffs they face during the research process instead of juggling with vague notions of unquantifiable philosophical principles.

> Of course, we see the relevance of Laken's work; we met in Eindhoven to discuss an early version of the paper.

Reviewer: ```suggestion
Of course, we see the relevance of Laken's work; we met in Eindhoven to discuss an early version of the paper.
With respect to how well developed our respective arguments are, we would like to point out that we think that a key part of our argument is the mathematical formulation that enables research to actually compute tradeoffs they face during the research process instead of juggling with vague notions of unquantifiable philosophical principles.
```

:::reviewer
The authors argue that one interpretation of preregistration is "the traditional application of preregistration to research paradigms that focus on confirmation by maximizing the theoretical risk or, equivalently, by limiting type-I error (when dichotomous decisions about theories are an inferential goal).
I would say it is better to argue preregistration aims to *control* the error rate, not maximize the true negative rate.
The authors then propose their own view: "the objective of preregistration is not the maximization of theoretical risk but rather the minimization of uncertainty about the theoretical risk."
I think the authors are in the right direction, but miss some things.
What the authors get right is that preregistration is about communicating something about the test to others.
:::

This is why we deliberately chose to use the term "uncertainty" and not "control".
Reducing uncertainty directly invokes the concept of communication.

> Control also seems to imply that it is a property of the procedure, as it seems curious to suggest that control differs between the reader and the original researcher (we argue that the uncertainty is lower for the latter as they have all the information).

Reviewer: I don't fully understand that.
